app/views/shared/_progress_bar.html.slim

```diff
@@ -0,0 +1,11 @@
+.govuk-grid-row
+  ol.progress-bar--container.govuk-list role="progressbar"
+
+    - bar.progress_bar_info.map do |item|
```

Comment:
here

Reply:
so `progress_bar_info` is our array and `item` is one og the key/value objects containing the info. The array is an object on `bar` which is our progress decorated as a bar. Naming here can be tweaked for clarity.

Reply:
You described something that I'd see as:
```ruby
bar.sections do |section|
```
but I like `increment` because it is how I'd articulate a progress bar advancing.

Reply:
Perhaps that more appropriate for the horizontal line...

Reply:
Sorry, i didnt see this. I called it `attributes` because i decided that's what they are, but happy to make a change if you think that's more appropriate